tests/components/device_automation/test_init.py

```diff
@@ -497,43 +520,54 @@ def _same_lists(a, b):
         {
             automation.DOMAIN: {
                 "alias": "hello",
-                "trigger": {"platform": "event", "event_type": "test_event1"},
-                "action": {"device_id": "", "domain": "test"},
+                "trigger": {
+                    "platform": "event",
+                    "event_type": "test_event1"
+                },
+                "action": {
+                    "device_id": "",
+                    "domain": "test"
+                },
             }
         },
     )
 
-    assert (
-        "Integration 'test' does not support device automation actions" in caplog.text
-    )
+    assert ("Integration 'test' does not support device automation actions" in
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)